### ISSUE TITLE
Bug in Sub-graph loss

### ISSUE THREAD

**Issue by fjchange** [frostinassiky/gtad#50]
https://github.com/frostinassiky/gtad/blob/6deb5b1bc6883b48bd22e0cc593069643c953e3d/gtad_lib/loss_function.py#L23

```
def subgraph_loss_func(pred_bm, gt_iou_map, bm_mask, lambda1=10):
    pred_bm_wcs = pred_bm[:, 0].contiguous()
    pred_bm_mse = pred_bm[:, 1].contiguous()
->  wce_loss = pem_reg_loss_func(pred_bm_wcs, gt_iou_map, bm_mask)
->  mse_loss = pem_cls_loss_func(pred_bm_mse, gt_iou_map, bm_mask)
    subgraph_loss = wce_loss + lambda1* mse_loss
    return subgraph_loss
```

Here, it is obvious that the wce_loss and mse_loss are wrongly named and lead to the wrong weighted sum.

**Comment by frostinassiky** [comment]
Very good catch @fjchange . This is a bug.
